src/redux/books/books.js

```diff
@@ -0,0 +1,29 @@
+import { createReducer } from '@reduxjs/toolkit';
+
+// Define action type for addding & removing
+const ADD_BOOK = 'bookstore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
+
+export const addBook = (book) => ({
+  type: ADD_BOOK,
+  payload: book,
+});
+
+export const removeBook = (bookId) => ({
+  type: ADD_BOOK,
+  payload: bookId,
+});
+
+// set intial state to an empty array of books
+const initialState = [];
+
+const booksReducer = createReducer(initialState, {
+  [ADD_BOOK]: (state, action) => {
+    state.push(action.payload);
```

Comment:
:red_circle: As requested by the previous reviewer you should avoid using mutate the state. Mutating your state would result in inconsistent UI. But besides that, there are some other consequences. It would [break time-travel debugging](https://blog.isquaredsoftware.com/2017/05/idiomatic-redux-tao-of-redux-part-1/#the-original-selling-point-redux-devtools), an essential development experience functionality that expects you to follow redux's best practices.

A screenshot of the previous reviewer's comment :arrow_down: 
![reviewer](https://user-images.githubusercontent.com/26736582/214283495-d3b3fe31-d146-49ea-aef1-cc005a4f8e88.png)

:red_circle:  :red_circle:  Kindly make sure you return the state as requested by the previous reviewer. The return keyword stops the execution of a function and returns the desired output.

**NOTE**: Because your arrow functions use curly brackets, {} you cannot implicitly return. To have an implicit return please make sure the function body is an expression, not a block